src/Integration.UnitTests/Binding/ProjectBindingOperationTests.cs

```diff
@@ -60,6 +60,7 @@ public void TestInitialize()
             this.serviceProvider.RegisterService(typeof(ISourceControlledFileSystem), this.sccFileSystem);
             this.serviceProvider.RegisterService(typeof(IRuleSetSerializer), this.ruleSetFS);
             this.serviceProvider.RegisterService(typeof(ISolutionRuleSetsInformationProvider), new SolutionRuleSetsInformationProvider(this.serviceProvider));
+            this.serviceProvider.RegisterService(typeof(IProjectSystemHelper), this.projectSystemHelper);
```

Comment:
The _IProjectSystemHelper_ is now required by more of the tests so it's now registered by default.